docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+bot:
+  build: .
+  links:
+    - redis
+    - db
+  volumes:
+    - ./:/go/src/github.com/BotBotMe/botbot-bot/
+
+redis:
+  image: redis:latest
+  ports:
+    - 6379
+
+db:
+  image: postgres:9.5
+  ports:
+    - "5432:5432"
+  volumes:
+      - ./init-db:/docker-entrypoint-initdb.d/
+      - ./sql:/pg-tmp/
```

Comment:
This won't persist the DB, right? Would people want to use docker beyond just testing?

Can we throw all Docker related stuff into a parent folder so it isn't cluttering the root of the repo?